docs/guides/token_guide.md

```diff
@@ -31,3 +27,124 @@ Below is an overview of how the migration will take place:
 - Your funds are successfully migrated.
 
 For further information on the migration process, see our [blog post](https://blog.iota.org/firefly-token-migration/).
+
+
+
+
+## Exchanges Migration Guide 
+
+To transfer your user's tokens securely to the new Chrysalis (IOTA 1.5) network, we provide this helpful guide. 
+
+There are potentially three ways with which you can migrate your funds from from old (legacy) to new (chrysalis) network:
+
+
+1. Via **Firefly**. This procedure is explained [in this blog entry](https://blog.iota.org/firefly-token-migration/). This guide will just cover the programmatically approach to migrate.
+
+2. Via a **Migration Address** Use wallet library solely for migrating funds from old to new network (in this case, you will have to generate and manage your new seed and migration address);
+
+3. Via a **legacy Seed**. Use wallet library for generating, managing your new seed and migration process;
+
+
+
+#### Migration Guide - Via an Migration Address
+
+With this scenario, you generate your recovery phrase yourself. Then generate an address from it, generate migration address (as in the example below) and then use the same steps performed in section 1 for migrating.
+
+With this approach, its very easy to migrate funds to the new Chrysalis network. You just generate a new Chrysalis address and convert it to the old Trinary Format. 
+If you send Funds to this migration address, it bridges the network and you have the funds available in the Chrysalis network.
+
+#### generate migration address
+This is an example, how to create a Migration Address with Node.js:
+
+```javascript=
+function run() {
+  const { AccountManager } = require('../lib')
+  const manager = new AccountManager()
+  let address = "atoi1qpheapgshz9saf8mc6zdly96xyz5qdc2qspsv7ezem6fw8lv869ms3amw0g";
+  let migration_address = manager.generateMigrationAddress(address);
+  console.log(migration_address)
+}
+run()
+```
+
+Code: https://github.com/iotaledger/wallet.rs/blob/develop/bindings/nodejs/tests/migration.js
+
+
+### Be safe!
+
+Please take care that you...
+- ... dont create outputs that are lower that 1 Mi (to prevent Dust Protection).
+- ... you use a limited amount of inputs (<10) and create small Bundles (otherwise your PoW will just takes to long).
+- ... use the [bundle-miner](https://github.com/iotaledger/iota.rs/tree/migration/iota-bundle-miner), if old bundle hashes include spent address. The bundle-miner will create bundles that reveal as few new parts of the private key as possible.
```

Comment:
`if old bundle hashes include spent address` should rather be
`if inputs include spent address` the bundle hashes are needed for the mining, but they don't contain the addresses